LivingCity/command_line_options.ini

```diff
@@ -12,3 +12,4 @@ NUM_PASSES=1
 TIME_STEP=0.5
 START_HR=5
 END_HR=12
+SHOW_BENCHMARKS=true
```

Comment:
When do we use this flag? I don't see any code (should be in `b18CommandLineVersion.cpp`) that does something based on this bool.

Reply:
It's [parsed](https://github.com/UDST/manta/blob/6004b904f12b9d4f6152e575f7e685bb4ef4fb6d/LivingCity/traffic/b18CommandLineVersion.cpp#L41), then it [sets up the Benchmarker settings](https://github.com/UDST/manta/blob/6004b904f12b9d4f6152e575f7e685bb4ef4fb6d/LivingCity/traffic/b18CommandLineVersion.cpp#L47) and then [each Benchmarker checks if the flag is true before printing](https://github.com/UDST/manta/blob/6004b904f12b9d4f6152e575f7e685bb4ef4fb6d/src/benchmarker.cpp#L35)

Reply:
Got it! Thanks.